csvkit/utilities/csvstat.py

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env python
 
 import codecs
-import warnings
-from collections import OrderedDict
+from collections import Counter, OrderedDict
 from decimal import Decimal
+import locale
+import warnings
 
 import agate
 import six
-from babel.numbers import format_decimal
```

Comment:
The `babel` library has a large upfront cost. I re-implemented this function using python's `locale` library.